roles/parse_leapp_report/README.md

```diff
@@ -8,7 +8,7 @@ The `parse_leapp_report` role is used by the `analysis` or `upgrade` roles to ch
 |-------------------------|-----------------------|-----------------------------------------------------|
 | result_filename         | "/var/log/leapp/leapp-report.txt" | Path of the Leapp pre-upgrade report file. |
 | result_filename_prefix  | "/var/log/leapp/leapp-report      | The path used and the prefix name setting for the Leapp report |
-| result_filename_json    | "{{ result_filename_prefix }}.json | JSON filename using the selected "result_filename_prefix" |
```

Comment:
Looks like you deleted this line, can you put it back because it doesn't look like it should be deleted then it looks good